### ISSUE TITLE
Accessibility - Add skip link to navigation

### ISSUE THREAD

**Issue by dmtrek14** [Call-for-Code-for-Racial-Justice/Legit-Info#135]
### Background on the problem the feature will solve/improved user experience
A 'skip' link allows for those using a keyboard to skip blocks of repeated content, such as a menu, so that they don't have to tab through all menu options after navigating to a page. This is part of the [WCAG guidelines 2.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html) and is further detailed [here](https://www.w3.org/WAI/standards-guidelines/act/rules/bypass-blocks-cf77f2/).

### Describe the solution you'd like

Add a skip link to the main menu that is hidden until it is tabbed to. This should have an anchor to the main element, so if main doesn't have an id, one should be added. Add css to keep the skip link hidden until focused and make sure the link is noticeable and comports with other styles and works on mobile devices, too.

### Tasks

- [ ] Add link to main navigation
- [ ] Add id to main if needed and make sure anchor of link points to this id
- [ ] Style the link

### Acceptance Criteria

When a user hits a page, they will not see the 'skip to main' link initially. But when the tab key is hit, this link will be highlighted/focused and be visible in the menu. When the skip link is hit (clicked or enter hit on keyboard), the user's focus should be shifted to main and when tab is next hit, the focus should go to the next focusable item inside main (a link, form input, or a button, for instance). Any styling of the skip link should be compliant with WCAG color contrast requirements.
